### PR TITLE
Return of the iOS target 

### LINKED ISSUE #16: Add an iOS framework target

### PR DESCRIPTION
**Pull request by tombooth**
I've had another go at setting up an iOS target that is started from the current master branch, but based on the work @phatblat did in #44.

This PR also switches the TravisCI build to use the *updated* `script/cibuild` file.

It also includes a switch to ZipArchive rather than using `unzip` directly so I could get the tests working against the iOS target.

Resolves #16

### CODE REVIEW

**Comment by phatblat** [comment]
Bravo! :clap:

**Comment by mdiep** [comment]
Thanks! It may take me a few days to review this, but I'm excited to see it!

File: SwiftGit2Tests/Fixtures/Fixtures.swift [hunk 1]
```diff
@@ -32,12 +33,18 @@ final class Fixtures {
...
+		#else
+			let platform = "iOS"
+		#endif
+		let bundleIdentifier = String(format: "org.libgit2.SwiftGit2-%@Tests", arguments: [platform])
+		let bundle = NSBundle(identifier: bundleIdentifier)!
 		let zipURLs = bundle.URLsForResourcesWithExtension("zip", subdirectory: nil)! as [NSURL]
 		
 		for URL in zipURLs {
-			unzipFileAtURL(URL, intoDirectoryAtURL: directoryURL)
+			SSZipArchive.unzipFileAtPath(URL.path!, toDestination: directoryURL.path!)
```
Comment (modocache) [inline SwiftGit2Tests/Fixtures/Fixtures.swift]: Awesome!

**Comment by modocache** [comment]
This all looks good to me, although I admit I didn't go over the `SwiftGit2.xcodeproj/project.pbxproj` changes line-by-line. It looks like the SwiftGit2-iOS tests passed, though, so I day ship it!

(By the way, if you update the Quick submodule reference to v0.9.1 or later, the test output will be much easier to read. :smile: )

File: SwiftGit2.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -739,12 +1179,227 @@
...
-			target = BEB31F8E1A0E563900F525B9 /* libgit2 */;
+			target = BEB31F8E1A0E563900F525B9 /* libgit2-OSX */;
 			targetProxy = BEB31F941A0E56E200F525B9 /* PBXContainerItemProxy */;
 		};
 /* End PBXTargetDependency section */
 
 /* Begin XCBuildConfiguration section */
+		621E66B21C72958800A0F352 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = BEB31F561A0D75EE00F525B9 /* Mac-Framework.xcconfig */;
```
Comment (mdiep) [inline SwiftGit2.xcodeproj/project.pbxproj]: The iOS framework should have the iOS framework xcconfig as its base configuration. The iOS test framework should also have the iOS version of the configuration selected.

configs

**Comment by mdiep** [comment]
Just the one note. Otherwise this looks great!

**Comment by tombooth** [comment]
I've made the change and rebased it into the structural changes commit. Excited to see this merged :shipit: !

**Comment by mdiep** [comment]
:tada:

**Pull request merged**